docs/manual.rst

```diff
@@ -1290,21 +1250,18 @@ Shapely's binary :meth:`within` predicate.
 
 .. code-block:: python
 
-  class Within:
-      def __init__(self, o):
-          self.o = o
-      def __lt__(self, other):
-          return self.o.within(other.o)
+  >>> class Within:
+  ...     def __init__(self, o):
+  ...         self.o = o
+  ...     def __lt__(self, other):
+  ...         return self.o.within(other.o)
 
 As the howto says, the `less than` comparison is guaranteed to be used in
-sorting. That's what we'll rely on to spatially sort, and the reason why we use
-:meth:`within` in reverse instead of :meth:`contains`. Trying it out on features
+sorting. That's what we'll rely on to spatially sort. Trying it out on features
 `d` and `c`, we see that it works.
 
 .. code-block:: pycon
 
-  >>> d < c
```

Comment:
Such comparison operator as shortcut for contains also didn't work in Shapely 1.7 or 1.8, not sure if that ever worked in the past